app/src/pages/Project/Card.tsx

```diff
@@ -221,32 +187,52 @@ const Card = ({ card }: ICardProps) => {
         onRemoveCard={onRemoveCard}
       />
       <div className='flex'>
-        <section className='h-[calc(100vh_-_112px)] flex-1 hide-scrollbar overflow-y-scroll overflow-x-hidden z-20 pt-[20px] pl-[30px] '>
-          <textarea
-            className='font-rubik align-middle h-auto min-h-[40px] text-[32px] border-none ml-12 mb-6 resize-none overflow-hidden w-[300px] md:w-3/4 lg:w-5/6'
-            rows={1}
-            placeholder='Type title'
-            ref={inputRef}
-            value={title}
-            onChange={onTitleChange}
-          />
-          <Editor key={card.id} projectId={projectId} card={card} users={users} />
-        </section>
-        <aside className='border-l-1 border-borders-purple w-[320px] lg:w-[400px] xl:w-[500px] 2xl:w-[600px]'>
+        {!isMobile && (
+          <section className='h-[calc(100vh_-_112px)] flex-1 hide-scrollbar overflow-y-scroll overflow-x-hidden z-20 pt-[20px] pl-[30px]'>
+            <textarea
+              className='font-rubik align-middle h-auto min-h-[40px] text-[32px] border-none ml-12 mb-6 resize-none overflow-hidden w-[300px] md:w-3/4 lg:w-5/6'
+              rows={1}
+              placeholder='Type title'
+              ref={inputRef}
+              value={title}
+              onChange={onTitleChange}
+            />
+            <Editor key={card.id} projectId={projectId} card={card} users={users} />
+          </section>
+        )}
+
+        <aside
+          className={clsx(
+            'border-l-1 border-borders-purple',
+            isMobile ? 'w-full' : 'w-[320px] lg:w-[400px] xl:w-[500px] 2xl:w-[600px]',
+          )}
+        >
           <CardTabs
             activeTab={activeTab}
             setActiveTab={setActiveTab}
             cardChatsLength={cardChats.length}
             unreadDiscussions={unreadDiscussions}
             filesLength={files.length}
           />
-          {
+          {isMobile && activeTab === 'editor' ? (
+            <div className='h-[calc(100vh_-_112px)] overflow-y-auto'>
+              <textarea
+                className='font-rubik align-middle h-auto min-h-[40px] text-[32px] border-none ml-12 mb-6 resize-none overflow-hidden w-[300px]'
+                rows={1}
+                placeholder='Type title'
+                ref={inputRef}
+                value={title}
+                onChange={onTitleChange}
+              />
+              <Editor key={card.id} projectId={projectId} card={card} users={users} />
+            </div>
+          ) : (
             {
               discussions: <Discussions users={users} />,
               attachments: <Attachments files={files} />,
-              // updates: <Updates />,
+              editor: null,
             }[activeTab]

```

Comment:
Put it in a separate component with conditional rendering, actually take a look `React.lazy` for these comp